homeassistant/components/zhong_hong/strings.json

```
@@ -0,0 +1,43 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up ZhongHong HVAC",
+        "description": "Configure connection to your ZhongHong HVAC gateway.\n\nDefault port is {default_port} and default gateway address is {default_gateway}.",
+        "data": {
+          "host": "Gateway IP Address",
+          "port": "Port",
+          "gateway_address": "Gateway Address"
+        }
+      },
+      "reconfigure": {
+        "title": "Reconfigure ZhongHong HVAC",
+        "description": "Update the connection settings for your ZhongHong HVAC gateway.",
+        "data": {
+          "host": "Gateway IP Address",
+          "port": "Port",
+          "gateway_address": "Gateway Address"
```

These duplicates should be replaced with references:

```suggestion
          "host": "[%key:component::zhong_hong::config::step::user::data::host%]",
          "port": "[%key:component::zhong_hong::config::step::user::data::port%]",
          "gateway_address": "[%key:component::zhong_hong::config::step::user::data::gateway_address%]"
```